application/src/main/java/org/togetherjava/tjbot/commands/free/Util.java

```diff
@@ -0,0 +1,27 @@
+package org.togetherjava.tjbot.commands.free;
+
+import net.dv8tion.jda.api.interactions.Interaction;
+import org.jetbrains.annotations.NotNull;
+
+import java.time.OffsetDateTime;
+import java.time.temporal.ChronoUnit;
+
+public class Util {
+    // private constructor to prevent this class getting instantiated
+    private Util() {}
+
+    /**
+     * Helper method to easily send ephemeral messages to users.
+     * 
+     * @param interaction The event or hook that this message is responding to
+     * @param message The text to be display for the user to read.
+     */
+    public static void sendErrorMessage(@NotNull Interaction interaction, @NotNull String message) {
+        interaction.reply(message).setEphemeral(true).queue();
+    }
+
+    public static @NotNull OffsetDateTime anHourAgo() {
```

Comment:
missing javadoc

Reply:
may i ask, why `OffsetDateTime`? why not `Instant`? Latter sounds more correct to me.

Reply:
Because the time stored in a message is OffsetDateTime and this is for comparison.

This is probably obsolete,
@Tais993 suggested a change that will fix 4 problems with the inactive method (which this is for) with only a small loss of accuracy that we dont really need.

Reply:
fair